atd-vzd/migrations/default/1697646059138_update_fatality_soft_delete_function/up.sql

```diff
@@ -0,0 +1,60 @@
+CREATE OR REPLACE FUNCTION public.update_fatality_soft_delete()
+ RETURNS trigger
+ LANGUAGE plpgsql
+AS $function$
+DECLARE
+    cris_id integer; -- used to hold vz id when triggered from cris_facts schema
+    injury_severity_id integer;
+    BEGIN
+    RAISE NOTICE 'Here we are.';
```

Comment:
is this like a stray console log :smile: also TIL, i love console logging for debugging so this is a helpful tip!!